packages/site/src/main.css

```diff
@@ -187,4 +187,10 @@ iframe {
   .contentCardWrapper {
     gap: var(--space-larger);
   }
+}
+
+.stickySectionHeader {
+  position: sticky;
+  top: 0;
+  z-index: 1;
```

Comment:
This is the "global" sticky styling, with the individual instances applying their own layout, background, etc.

This _might_ not be worth sharing, and I could style them in situ if preferred.